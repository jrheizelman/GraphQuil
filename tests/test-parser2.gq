int func1() {
 bool be;
 be = false;
}

int func2() {
print("hi");
}

int main()
{
func1();
func2();
}
