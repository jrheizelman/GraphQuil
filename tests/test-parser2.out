int main()
{
func1();
func2();
}

int func2()
{
print(hi);
}

int func1()
{
bool be;
be = false;
}